### ISSUE TITLE
Title subtext does not meet WCAG requirement

### ISSUE THREAD

**Issue by rosecraftdesigns** [Mindfulnessviking/Aeltherra#1]
The subtext below the title on the homepage does not pass the WCAG contrast text as it has failed the small text test. Maybe will need to consider adding shadow to the text like the title to improve contrast. 

<img width="495" height="111" alt="Image" src="https://github.com/user-attachments/assets/6eada406-acec-4483-ba63-4873235e1ba0" />
